### PR TITLE
U4X-827: Improve the frontend builds to include the latest images

### PR DESCRIPTION
**Pull request by slubwama**
https://metsprogramme.atlassian.net/browse/U4X-827

### CODE REVIEW

File: tools/src/main/resources/frontendconfigs/build.sh [hunk 1]
```diff
@@ -5,24 +5,27 @@ echo "Cleaning up previous build artifacts ..."
...
+npx --legacy-peer-deps openmrs@latest assemble \
   --manifest \
   --mode config \
   --config ./configuration/spa-build-config.json \
   --target ./frontend
 # Build assets
 echo "Building assets for UgandaEMR+..."
 CWD=$(pwd)
-npx --legacy-peer-deps openmrs@5.8.1 build \
+npx --legacy-peer-deps openmrs@latest build \
```
Comment (akileng56) [inline tools/src/main/resources/frontendconfigs/build.sh:16]: openmrs@latest can easily change to a version not tested, hence breaking changes when you rebuild

File: tools/src/main/resources/frontendconfigs/build.sh [hunk 2]
```diff
@@ -5,24 +5,27 @@ echo "Cleaning up previous build artifacts ..."
 rm -rf frontend
 # Assemble assets
 echo "Assembling assets ..."
-npx --legacy-peer-deps openmrs@5.8.1 assemble \
+npx --legacy-peer-deps openmrs@latest assemble \
```
Comment (akileng56) [inline tools/src/main/resources/frontendconfigs/build.sh:8]: @slubwama i think lets leave this at a version than latest

**Review by akileng56 — CHANGES_REQUESTED** [review]

File: tools/src/main/resources/frontendconfigs/build.sh (hunk 2 above)
Comment (slubwama) [reply tools/src/main/resources/frontendconfigs/build.sh:8]: @akileng56  that would be hard to build for anyone without the selected version in their local npm. They have to download it manually. To prove this delete your .npm  directory and try to run mvn clean install on this project. 

So opting for latest eliminates that. I also think that for now 5.8.1 is the latest for devtools and core which should be around for over a month

File: tools/src/main/resources/frontendconfigs/build.sh (hunk 1 above)
Comment (slubwama) [reply tools/src/main/resources/frontendconfigs/build.sh:16]: @akileng56 that would be hard to build for anyone without the selected version in their local npm. They have to download it manually. To prove this delete your .npm directory and try to run mvn clean install on this project.

So opting for latest eliminates that. I also think that for now 5.8.1 is the latest for devtools and core which should be around for over a month

**Review by akileng56 — APPROVED** [review]

**Pull request merged**